data/beaconrestapi/src/test/java/tech/pegasys/teku/beaconrestapi/handlers/v1/node/GetSyncingTest.java

```diff
@@ -49,13 +49,26 @@ public void shouldGetSyncingStatusSyncing() throws Exception {
   public void shouldGetSyncStatusInSync() throws Exception {
     when(syncService.getSyncStatus()).thenReturn(getSyncStatus(false, 1, 10, 11));
     when(syncService.getCurrentSyncState()).thenReturn(SyncState.IN_SYNC);
+    when(executionClientDataProvider.isExecutionClientAvailable()).thenReturn(true);
 
     handler.handleRequest(request);
     assertThat(request.getResponseCode()).isEqualTo(SC_OK);
     assertThat(request.getResponseBody())
         .isEqualTo(new GetSyncing.SyncStatusData(false, false, 10, 0));
   }
 
+  @Test
+  public void shouldGetSyncStatusSyncingWhenExecutionLayerIsOffline() throws Exception {
+    when(executionClientDataProvider.isExecutionClientAvailable()).thenReturn(false);
+    when(syncService.getSyncStatus()).thenReturn(getSyncStatus(false, 1, 10, 11));
+    when(syncService.getCurrentSyncState()).thenReturn(SyncState.IN_SYNC);
+
+    handler.handleRequest(request);
+    assertThat(request.getResponseCode()).isEqualTo(SC_OK);
+    assertThat(request.getResponseBody())
+        .isEqualTo(new GetSyncing.SyncStatusData(true, false, 10, 1));
+  }
+
```

Comment:
this test is probably not adequate if the handler test is to be believed, we may need an acceptance test that shuts down the EL and shows the result...

Reply:
OK. Actually I run some tests locally and hesitated to add an acceptance test. Will do